README.md

```diff
@@ -0,0 +1,178 @@
+<a name="readme-top"></a>
+
+<div align="center">
+   <br/>
+
+  <h3><b>Expense App</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 About the Project ](#-about-the-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+# 📖 About the Project <a name="about-project"></a>
+
+**Expense App** is an transaction tracker app containing data about transactions based on categories. A user can register/login and create transactions to track them.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Databases</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Contains data about categories**
+- **Contains data about transactions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Link to Live demo]()
+- [Quick Overview]()
```

Comment:
- [ ] As stated in the project requirement kindly make sure that you record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:
- a description of the project.
- a demo of the project features.
- you should also highlight some interesting piece of code or something you built that you are very proud of. And Provide the link to the video on Readme File

- [ ] As stated in the project requirement kindly make sure that you deploys the application using Render, Netlify or GitHub Pages.The project is deployed properly, the home and detail page works as expected. Kindly Provide A link to the deployed project is present in the README.
- Both for the Project overview (Video) and Live demo provide the  Links .